analysis_options.yaml

```diff
@@ -12,6 +12,7 @@ analyzer:
     missing_return: warning
   exclude:
     - build/**
+    - flutter_packages/**
```

Comment:
I'm curious why this is excluded (is this dir being created somewhere)?

Reply:
Yes, it is legacy from removed step. Removed. Thanks.